src/Razor/test/testassets/ClassLibraryMvc21/ClassLibraryMvc21.csproj

```diff
@@ -14,5 +14,6 @@
       <!-- Avoid exporting types from real MVC that will conflict with the test shim -->
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
To confirm: This and SimpleMvc21NetFx.csproj` don't import ResolveReferences.targets❔ If that's the case, this is fine.

Reply:
Correct